### PR TITLE
{lib}[foss/2020a] preCICE v2.1.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/762272ed245fc82be192e2dfb26f614f for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5e179d1ff165defabc3add8b0946b206 for a full test report.

File: easybuild/easyconfigs/p/preCICE/preCICE-2.1.1-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+is a coupling library for partitioned multi-physics simulations,
+including, but not restricted to fluid-structure interaction and
+conjugate heat transfer simulations. Partitioned means that preCICE
+couples existing programs (solvers) capable of simulating a subpart of
+the complete physics involved in a simulation. This allows for the high
+flexibility that is needed to keep a decent time-to-solution for complex
+multi-physics scenarios."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+# Needed to make the PETSc testing works.
```
Comment (smoors) [inline easybuild/easyconfigs/p/preCICE/preCICE-2.1.1-foss-2020a-Python-3.8.2.eb:18]: nit:
```suggestion
# Needed to make the PETSc testing work.
```

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3134.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4c7f83e036d27e6281350d353fed1e05 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2707.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/c1f716028cb70c2fb7d7f6bba12d9fde for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 4 out of 5 (1 easyconfigs in total)
node320.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/4a05ac44d362454457f54db9fe42a330 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/2cff2aa4d587e687d1b08a0b91671779 for a full test report.

**Comment by boegel** [comment]
@akesandgren 
```
  Could NOT find Eigen3 (missing: EIGEN3_VERSION_OK) (Required is at least version "3.2")
  ```

**Comment by akesandgren** [comment]
Then that Eigen install need to be rebuilt after the easyblock/config was fixed.
See commit bb3c1ec4edbebeaa8db1cfa26ba2442b79c0779e in easyblocks

**Comment by boegel** [comment]
@boegelbot please test @ generoso

(I did a forced reinstall of `Eigen-3.3.7-GCCcore-9.3.0.eb` in the installations used by `boegelbot`)

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
node370.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/6c378b1d6410c62473d44baf8902584e for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @akesandgren!

**Pull request merged**